docs/README.md

```diff
@@ -1,60 +1,68 @@
-📢 Don't fork this project. Use, [contribute](https://github.com/vtex-apps/awesome-io#contributing), or open issues through [Store Discussion](https://github.com/vtex-apps/store-discussion).
+📢 Use this project, [contribute](https://github.com/vtex-apps/breadcrumb) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

Comment:
📢 Use this project, contribute to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).